ros_buildfarm/templates/status/js/setup.js

```diff
@@ -238,15 +238,24 @@ function filter_table() {
       result_rows = $.map(window.rows, function(row) {
         for (var i = 0; i < queries.length; i++) {
           var is_known_query = false;
+          var is_id_query = false;
           for(var q in QUERY_TRANSFORMS) {
-            if (queries[i] === QUERY_TRANSFORMS[q]) {
+            if (RegExp("^("+QUERY_TRANSFORMS[q]+")$").test(queries[i])) {
```

Comment:
yeah, I did it this way to keep it compatible with the other status pages that are using setup.js but perhaps it makes sense to duplicate the file and make it specific to the blocked repos page (which is only using regex queries)